types/decentlab/PR-26/default.schema.json

```diff
@@ -1,23 +0,0 @@
-{
```

Comment:
why delete this?

Reply:
i don't see what it was but i guess for formatting purposes